### PR TITLE
Added support for RCON on Velocity servers

### PR DESCRIPTION
**Pull request by WhiteHoodHacker**
BungeeCord plugins, such as [orblazer/bungee-rcon](https://github.com/orblazer/bungee-rcon), are not directly compatible with Velocity servers.  As a result, the RCON plugin fails to function on Velocity servers.

I suggest using another third-party plugin, [UnioDex/VelocityRcon](https://github.com/UnioDex/VelocityRcon), to bring RCON support for Velocity servers. I haven't found any problems using it in my tests and I also audited the source code for issues.

### CODE REVIEW

**Review by itzg — COMMENTED** [review]
Changes looked like they were good, but I'm seeing an error during sed step:

```
> docker run -it --rm -e EULA=TRUE -e TYPE=VELOCITY -e ENABLE_RCON=true bungee
WARNING: The requested image's platform (linux/arm/v7) does not match the detected host platform (linux/amd64) and no specific platform was requested
Resolving type given VELOCITY
Downloading https://versions.velocitypowered.com/download/latest.jar
Downloading Velocity rcon plugin
Copy Velocity rcon configuration
/usr/bin/sed: can't read /tmp/rcon-velocity-config.toml: No such file or directory
/usr/bin/sed: can't read /tmp/rcon-velocity-config.toml: No such file or directory
/usr/bin/mv: cannot stat '/tmp/rcon-velocity-config.toml': No such file or directory
```

**Comment by itzg** [comment]
...oh it's missing a `COPY` of the config file like

https://github.com/itzg/docker-bungeecord/blob/6dab2e8edbf1846fc911ae1644b9c1895ac3b43f/Dockerfile#L42

however, now that I see this again, I don't like that the config file is getting staged into a `/tmp` path. It would be great if you could change yours and the existing to a `/templates` path.

**Comment by WhiteHoodHacker** [comment]
Ah oops, I had forgotten to update the docker file from my development environment. I'll commit your suggested changes in a bit.

**Review by itzg — COMMENTED** [review]
Thanks for changing. Getting further, but now seeing this during startup:

```
[18:50:23 INFO]: Loading plugins...
[18:50:23 ERROR]: Can't create plugin velocityrcon
com.google.inject.ProvisionException: Unable to provision, see the following errors:

1) Error injecting constructor, java.lang.ClassCastException: class java.lang.Long cannot be cast to class java.lang.String (java.lang.Long and java.lang.String are in module java.base of loader 'bootstrap')
  at me.uniodex.velocityrcon.VelocityRcon.<init>(VelocityRcon.java:43)
  at com.velocitypowered.proxy.plugin.loader.java.VelocityPluginModule.configure(VelocityPluginModule.java:48)
  while locating me.uniodex.velocityrcon.VelocityRcon
```

The rendered config file looks like:
```
root@9e4728fc64c4:/server/plugins/velocityrcon# cat rcon.toml
rcon-port = 25575
rcon-password =
rcon-colored = false
```

**Comment by itzg** [comment]
Maybe it's a newline thing because along with adding `-e RCON_PASSWORD=testing` I see that "line 2" got merged along with line 3:

```
Caused by: java.lang.RuntimeException: java.lang.IllegalStateException: Invalid value on line 2: rcon-password = testing
rcon-colored = false
```

**Comment by WhiteHoodHacker** [comment]
Oh, the issue is the values need to be wrapped in quotation marks.

File: rcon-config.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Port you want RCON to listen
-port: ${PORT}
+port: "${PORT}"
```
Comment (itzg) [inline rcon-config.yml:2]: Don't you need to be changing `rcon-velocity-config.toml`?

**Review by itzg — APPROVED** [review]
Works great now. Thanks.

**Comment by itzg** [comment]
This is now pushed to Docker Hub

**Pull request merged**